src/checkpoint/checkpoint_private.h

```diff
@@ -0,0 +1,15 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ * All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+#define WT_CHECKPOINT_SESSION_FLAGS (WT_SESSION_CAN_WAIT | WT_SESSION_IGNORE_CACHE_SIZE)
```

Comment:
Maybe this will grow someday, but having this one macro as the only private thing is sort of lonely.

Reply:
It will grow soon! Yury is working on the next ticket that will add stuff there: WT-13782